flake.nix

```
@@ -32,6 +32,26 @@
           pre-commit.settings.hooks.rustfmt.enable = true;
           pre-commit.settings.settings.rust.cargoManifestPath = "./rust/Cargo.toml";
 
+          # Check that we're using ///-style doc comments in Rust code.
+          #
+          # Unfortunately, rustfmt won't do this for us yet - at least not
+          # without nightly, and it might do too much.
+          pre-commit.settings.hooks.rust-doc-comments = {
+            enable = true;
+            files = ".*.rs$";
```

```suggestion
            files = "\\.rs$";
```

I think `.*` is redundant since you're not starting with `^`